protocol-configuration/mainnetAddresses.ts

```diff
@@ -855,6 +889,24 @@ export const MainnetContractsConfig = {
     category: AddressCategory.External
   },
 
+  aaveCompaniesMultisig: {
+    artifactName: 'unknown',
+    address: '0xe8d4D93D9728bD673b0197673a230F62255C7846',
+    category: AddressCategory.External
+  },
+
+  aaveCompaniesDaiEscrowMultisig: {
+    artifactName: 'unknown',
+    address: '0xA27A83A2433EbEB9f658B4420074B454846e1C5B',
+    category: AddressCategory.External
+  },
+
+  escrowedAaveDaiPCVDeposit: {
+    artifactName: 'ERC20PCVDepositWrapper',
+    address: '0x82c55A1Ab5C6F4b8e162b7dE24b50A38E1aFd38f',
+    category: AddressCategory.External
```

Comment:
This is not external its PCV. While you are at it, can you move the Deprecated contracts down into the deprecated section